### ISSUE TITLE
Problemas de desempenho no tablet A7 Samsung

### ISSUE THREAD

**Issue by TheMartinfer22** [olmps/memo#186]
Olá evidenciei um problema ao abrir o aplicativo, o mesmo demonstra perda de performance, somente na guia principal e logo após permanecer um tempo é fechado caso o usuário não realiza alguma ação.

Modelo do Tablet: **Samsung Galaxy Tab A7 SM-T505N**
Versão do aplicativo: **0.1.0 | 16 de julho 2021**
Observações adicionais: **Quando eu tento entrar em outra aba ele demora um pouco mas logo entra e o desempenho fica normal.**

**Comment by ggirotto** [comment]
Olá @TheMartinfer22 , obrigado pelo feedback.

Esse problema pode estar diretamente relacionado à issue https://github.com/olmps/memo/issues/162 que foi corrigido no PR https://github.com/olmps/memo/pull/185. Vamos subir uma atualização para a loja nos próximos dias com essa correção. Assim que ela estiver disponível te marcamos aqui para você realizar os testes novamente, pode ser?

**Comment by TheMartinfer22** [comment]
Claro, vou ficar atento as alterações! Então vou testar e voltar aqui para dar feedback.
